bin/run-tests.js

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env node
+
+var RSVP  = require('rsvp');
+var spawn = require('child_process').spawn;
+var chalk = require('chalk');
+var packages = require('../lib/packages');
+var runInSequence = require('../lib/run-in-sequence');
+
+function shouldPrint(inputString) {
+  var skipStrings = [
+    "*** WARNING: Method userSpaceScaleFactor",
+    "CoreText performance note:",
+  ];
+
+  for (var i = 0; i < skipStrings.length; i++) {
+    if (inputString.indexOf(skipStrings[i])) {
+      return false;
+    }
+  }
+
+  return true;
+}
+
+function run(queryString) {
+  return new RSVP.Promise(function(resolve, reject) {
+    var args = [
+      'bower_components/qunit-phantom-runner/runner.js',
+      './dist/tests/index.html?' + queryString
+    ];
+
+    console.log('Running: phantomjs ' + args.join(' '));
+
+    var child = spawn('phantomjs', args);
+    var result = {output: [], errors: [], code: null};
+
+    child.stdout.on('data', function (data) {
+      var string = data.toString();
+      var lines = string.split('\n');
+
+      lines.forEach(function(line) {
+        if (line.indexOf('0 failed.') > -1) {
+          console.log(chalk.green(line));
+        } else {
+          console.log(line);
+        }
+      });
+      result.output.push(string);
+    });
+
+    child.stderr.on('data', function (data) {
+      var string = data.toString();
+
+      if (shouldPrint(string)) {
+        result.errors.push(string);
+        console.error(chalk.red(string));
+      }
+    });
+
+    child.on('close', function (code) {
+      result.code = code;
+
+      if (code === 0) {
+        resolve(result);
+      } else {
+        reject(result);
+      }
+    });
+  });
+}
+
+var testFunctions = [];
+var emberChannel = process.env.EMBER_CHANNEL || 'release';
+
+testFunctions.push(function() {
+  return run('emberchannel=' + emberChannel);
+});
+testFunctions.push(function() {
+  return run('dist=min&emberchannel=' + emberChannel);
+});
+testFunctions.push(function() {
+  return run('dist=prod&emberchannel=' + emberChannel);
+});
```

Comment:
@rwjblue I'm trying to setup `npm test` to run against `debug`, `prod` and `min` versions of the code.

 `ember build` is only outputting the `ember-data.debug.js`. It's skipping the `.prod` and `.min` versions of both `ember-data` and `ember-data-tests`.

I'm using [this version](https://github.com/emberjs/emberjs-build/blob/f801dc6e1e1ef286d49882e041f092a32252d9d6/lib/ember-build.js#L60-L66) of `ember-build`.

They exist when I run `ember build --environment production`, which is fine for CI where broccoli's environment is `production`, but locally only the first test run will work.

ember proper doesn't seem to be doing anything differently from the `ember-build` perspective.

Any ideas?